server-kotlin/src/main/kotlin/city_sync/model/CityExport.kt

```diff
@@ -16,7 +17,7 @@ data class CityExport(
     val email: String?,
     val ico: String?,
     val dataBox: String?,
-    val eDesky: Short?,
+    val edesky: Short?,
```

Comment:
Nemenil bych pojmenovani, bez ohledu na spravnost, mame vsude (dataset, vyhledavani) eDesky.

Reply:
V tom export JSONu to má být s velkým nebo malým D?